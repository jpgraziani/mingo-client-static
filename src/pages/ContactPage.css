.contact-container {
  background-color: var(--deep-rose);
  width: 100vw;
}

.contact-form {
  display: flex;
  flex-direction: column;
  flex-wrap: wrap;
  margin: 0 auto;
  justify-content: center;
  align-items: center;
  padding: 4rem 0;
  color: var(--white);
}

.contact-form hr {
  margin: 0;
}

.contact-form h2 {
  padding: 0;
  margin: 0;
  line-height: 3.5rem;
  font-size: 2.5rem;
  color: white;
}

.field-user-email, 
.field-user-message {
  margin-top: 3rem;
}

#user-name, 
#user-email, 
#user-message {
  border: none;
  border-bottom: 2px solid var(--white);
  width: 80vw;
  background: none;
  color: var(--white);
  height: 50px;
}

#user-name {
  height: 50px;
}

#user-email {
  height: 50px;
}

#user-message {
  height: 175px;
}
.contact-container textarea {
  resize: none;
  font-size: 1.5rem;
  resize: none;
  
}

.contact-container textarea::placeholder {
  color: rgba(250, 250, 250, 0.6);
}

.contact-container input {
  resize: none;
  font-size: 1.5rem;
  resize: none;
}

.contact-container input::placeholder {
  color: rgba(250, 250, 250, 0.6);
}

.contact-container button {
  display: inline-block;
	margin-top: 3.5rem;
	margin-right: 1rem;
	padding: .9rem 1rem;
	text-decoration: none;
	font-size: 1.5rem;
	letter-spacing: 1px;
	font-weight: 600;
  border: 2px solid var(--white);
  background: none;
  color: var(--white);
}

.contact-container button:hover {
  border: 2px solid white;
  background: white;
  color: var(--deep-rose);
}

@media (min-width:769px) {
  #user-name, 
  #user-email, 
  #user-message {
    max-width: 800px; 
  }
}