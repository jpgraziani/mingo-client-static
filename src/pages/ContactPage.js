import React from 'react';

export default function ContactPage() {
  return (
    <div>
      <h2>Message Me</h2>
      <form action="https://formspree.io/f/xzbkrnjz" method="POST">
        <label htmlFor='user-name'>Name</label>
        <input id='user-name' name='name' type='text' placeholder='enter full name' required/> <br />
        <label htmlFor='user-email'>Email</label>
        <input id='user-email' name='email' type='text' placeholder='enter email' required /> <br />
        <label htmlFor='user-message'>Message</label>
        <textarea id='user-message' name='message' placeholder='enter message' required /> <br />
        <button type='submit'>submit</button>
      </form>
    </div>
  );
}