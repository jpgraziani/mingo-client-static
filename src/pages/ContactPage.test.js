import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import ContactPage from './ContactPage'

describe(`ContactPage component`, () => {
  it('renders an ContactPage by default', () => {
    const wrapper = shallow(<ContactPage />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
