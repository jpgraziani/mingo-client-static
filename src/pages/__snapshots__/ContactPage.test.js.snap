// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactPage component renders an ContactPage by default 1`] = `
<div>
  <h2>
    Message Me
  </h2>
  <form
    action="https://formspree.io/f/xzbkrnjz"
    method="POST"
  >
    <label
      htmlFor="user-name"
    >
      Name
    </label>
    <input
      id="user-name"
      name="name"
      placeholder="enter full name"
      required={true}
      type="text"
    />
     
    <br />
    <label
      htmlFor="user-email"
    >
      Email
    </label>
    <input
      id="user-email"
      name="email"
      placeholder="enter email"
      required={true}
      type="text"
    />
     
    <br />
    <label
      htmlFor="user-message"
    >
      Message
    </label>
    <textarea
      id="user-message"
      name="message"
      placeholder="enter message"
      required={true}
    />
     
    <br />
    <button
      type="submit"
    >
      submit
    </button>
  </form>
</div>
`;
